backend/src/models/reviews.go

```diff
@@ -47,38 +47,37 @@ func GetReviews(ctx context.Context, review *Review, following *[]Follows, sort
 		return nil, err
 	}
 
-	var query string
-	var queryArgs interface{}
-	if len(review.AlbumID) != 0 {
-		query = "album_id = ?"
-		queryArgs = review.AlbumID
-	} else if len(review.Username) != 0 {
-		if following != nil {
-			query = "username IN (?)"
-			followingUsernames := make([]string, len(*following))
-			for i, f := range *following {
-				followingUsernames[i] = f.Following
-			}
-			queryArgs = followingUsernames
-		} else {
-			query = "username = ?"
-			queryArgs = review.Username
+	prepend := ""
+	if sort == "popular" {
+		prepend = "reviews."
+	}
+
+	query := make(map[string]interface{})
```

Comment:
using a map is pretty huge brain, definitely cleaner than what i had lol

Reply:
i only do what chatgpt tells me